sox.dialog.js

```diff
@@ -89,40 +90,36 @@
         if (featureSettings) {
           const $settingsDiv = $('<div/>', {
             id: 'feature-settings-' + name,
-            'class': 'sox-feature-settings',
+            class: 'sox-feature-settings',
```

Comment:
`class` is a reserved word in JS so it's generally preferred to quote a key if it is called `class`. Whilst this will work, I think it's safer to just keep it quoted.